collector.go

```diff
@@ -0,0 +1,77 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package datadog
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"go.opencensus.io/stats/view"
+)
+
+const rate = float64(1)
+
+// client implements datadog.Client
+type collector struct {
+	opts Options
```

Comment:
Apologies for being so nit-picky, but I think we could remove some of the white space here, may I recommend:
```go
type collector struct {
    opts     Options
    client   *statsd.Client
    mu       sync.Mutex // guards viewData
    viewData map[string]*view.Data
}
```
I realize this is a nit, so feel free to disregard this comment if you prefer your version. Your code is very easy to follow and the variable naming is quite good, so it's easy to understand what everything is.